### DISCUSSION TITLE
Diversity and Inclusion: creating metrics

### DISCUSSION THREAD

**Discussion by GeorgiaHCA** [alan-turing-institute/AutSPACEs#366] [General]
We need metrics to ensure that we are reaching and empowering a diverse group. 

- [ ] Compile list of groups who could support diversity in project and help identify gaps 
- [ ] Onboard collaborators
- [ ] Co-design metrics
- [ ] Publish metrics on GitHub repo
- [ ] Measure project in relation to metrics 

(Could also potentially create a workshop with Insight group members?)

**Comment by EllenDevereux** [comment]
Please reach out to Autistica to complete this. Create a new issue for anything you need Fujitsu support with please.
